app/components/paypal-button.js

```diff
@@ -0,0 +1,46 @@
+import paypal from 'npm:paypal-checkout';
+import Component from '@ember/component';
+
+export default Component.extend({
+
+  async didInsertElement() {
+    this._super(...arguments);
+    let order = this.get('data');
+    let loader = this.get('loader');
+    let createPayload = {
+      'data': {
+        'attributes': {
+          'return-url' : `http://localhost:4200/orders/${order.identifier}/placed`,
+          'cancel-url' : `http://localhost:4200/orders/${order.identifier}/placed`
```

Comment:
Isn't it supposed to be different ? I mean both the `return-url` and the `cancel-url` are same.

Reply:
@srv-twry I am not sure how this thing will behave once the order is confirmed/cancelled. I added them as same because i am thinking of handling the response on the same route. I will change this if this does not behave as expected.